enterprise/validators.py

```diff
@@ -0,0 +1,24 @@
+"""
+Database models field validators.
+"""
+import os
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+def validate_image_extension(value):
+    """
+    Validate that a particular image extension.
+    """
+    ext = os.path.splitext(value.name)[1]  # [0] returns path+filename
+    valid_extensions = ['.png', ]
+    if not ext.lower() in valid_extensions:
+        raise ValidationError(u'Unsupported file extension.')
```

Comment:
@asadiqbal08 nit: `u'Unsupported file extension.'` should be marked for translation.

Also: `u` should not be necessary when `unicode_literals` import is added.